src/thunderbots/software/ai/intent/intent.h

```diff
@@ -3,7 +3,12 @@
 #include <string>
 
 // TODO: should this be an enum instead?
```

Comment:
I think we can just removed this `TODO`? (or if not, it should be associated with an issue)